rust/src/cli/firmware/realtek.rs

```diff
@@ -0,0 +1,265 @@
+// Copyright 2023 Google LLC
```

Comment:
maybe this could be named `rtk.rs` to be consistent with `drivers/rtk.rs` ?

Reply:
Done